cloud-consumption/src/test/java/com/sequenceiq/consumption/util/CloudStorageLocationUtilTest.java

```diff
@@ -0,0 +1,36 @@
+package com.sequenceiq.consumption.util;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import javax.validation.ValidationException;
+
+import org.junit.jupiter.api.Test;
+
+import com.sequenceiq.common.model.FileSystemType;
+
+public class CloudStorageLocationUtilTest {
+
+    private static final String S3_OBJECT_PATH = "s3a://bucket-name/folder/file";
+
+    private static final String WASB_OBJECT_PATH = "wasb://bucket-name/folder/file";
+
+    @Test
+    public void testGetBucketName() {
+        assertEquals("bucket-name", CloudStorageLocationUtil.getBucketName(FileSystemType.S3, S3_OBJECT_PATH));
+
+        assertThrows(ValidationException.class, () -> CloudStorageLocationUtil.getBucketName(FileSystemType.S3, WASB_OBJECT_PATH));
+    }
+
+    @Test
+    public void testValidateCloudStorageType() {
+        assertDoesNotThrow(() -> CloudStorageLocationUtil.validateCloudStorageType(FileSystemType.S3, S3_OBJECT_PATH));
+
+        assertThrows(ValidationException.class, () -> CloudStorageLocationUtil.getBucketName(FileSystemType.S3, WASB_OBJECT_PATH));
```

Comment:
This and the below assertions are separate test cases.
However their testing could be generalized with ParameterizedTest.